scripts/build.d/qt

```diff
@@ -16,21 +25,34 @@ if [ -z "${SYNCGIT}" ]; then
 
   download_md5 $pkgfn $pkgurl $pkgmd5
 
-  mkdir -p ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
-  pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src > /dev/null
-    tar Jxf ${DEVENVDLROOT}/$pkgfn
+  mkdir -p ${QTSRC}
+  if [ -d ${QTSRC}/$pkgfull ];
+  then
+    echo "QTSRC: ${QTSRC}/$pkgfull already exist, skip extract"
```

Comment:
I want an environment variable to control this behavior.  It should work like `SYNCGIT`.  Users can set the environment variable from outside to skip or force the extract.

Reply:
I see... I'll add the controll asap.